extensions/amp-story/1.0/amp-story.css

```diff
@@ -286,13 +287,12 @@ amp-story-grid-layer {
 }
 
 .i-amphtml-story-grid-template-fill > :first-child {
-  bottom: 0 !important;
-  display: block !important;
-  height: auto !important;
+  top: 0 !important;
   left: 0 !important;
+  height: 100% !important;
+  width: 100% !important;
```

Comment:
This will be overridden by the `width: auto` below, and will, from what I understand, break the layout. Could you please upload your test cases?

Reply:
Woops. In this case `width: auto` and `width: 100%` were behaving the same, though.

The tests can be found under the visual diff examples:

http://localhost:8000/examples/visual-tests/amp-story/amp-story-grid-layer-template-fill.html
http://localhost:8000/examples/visual-tests/amp-story/amp-story-grid-layer-template-thirds.html
http://localhost:8000/examples/visual-tests/amp-story/amp-story-grid-layer-template-horizontal.html
http://localhost:8000/examples/visual-tests/amp-story/amp-story-grid-layer-template-vertical.html

Reply:
I'm not actually sure we need to change `fill` layers at all, since the implementation doesn't use `grid-area`s.

Reply:
@newmuis you were right and the bug only affects to classes using `grid-area`s. Updated the PR to only update the thirds template's height.